rules/catch-error-name.js

```diff
@@ -31,7 +32,7 @@ const create = context => {
 
 	const options = {
 		name: 'error',
-		caughtErrorsIgnorePattern: /^_$|^[\dA-Za-z]+(e|E)rror$/.source,
+		caughtErrorsIgnorePattern: /^[\dA-Za-z]+[e|E]rror$/.source,
```

Comment:
L47 already ignored `_`

Reply:
See https://github.com/sindresorhus/eslint-plugin-unicorn/issues/648 , will fix it later.